presto-parser/src/main/java/com/facebook/presto/sql/tree/Cube.java

```diff
@@ -55,7 +55,7 @@ private Cube(Optional<NodeLocation> location, List<QualifiedName> columns)
     public List<Set<Expression>> enumerateGroupingSets()
     {
         return ImmutableList.copyOf(Sets.powerSet(columns.stream()
-                .map(QualifiedNameReference::new)
+                .map(DereferenceExpression::from)
```

Comment:
Possibly. The spec doesn't allow anything other than column names, but we might want to extend that.